src/Contexts/NetworkManagerContext.tsx

```diff
@@ -89,6 +89,15 @@ interface NetworkManagerContext {
 
   setTransferTxHash: (input: string) => void;
   transferTxHash: string;
+
+  setDepositRecipient: (input: string | undefined) => void;
+  depositRecipient: string | undefined;
+
+  setDepositAmount: (input: number | undefined) => void;
+  depositAmount: number | undefined;
+
+  setFallbackInitialized: (input: boolean) => void;
```

Comment:
When do you set it to `false`?

Reply:
Implemented fallback mechanism stop on transaction complete and abort.